### ISSUE TITLE
editMessageMedia does not function

### ISSUE THREAD

**Issue by zhtaihao** [kotlin-telegram-bot/kotlin-telegram-bot#213]
**Expected behavior**
[editMessageMedia](https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/main/telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt#L1220) edits the media of a message. The user sees this change immediately.

**Actual behavior**
Nothing happens.

**To Reproduce**
Call the method linked above and provide a bot-unique file ID.

**Screenshots/Traceback**

**Possible solutions**
Perhaps this is related to the JSON serialisation of the TelegramFile.ByFileId object? [[reference]](https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/main/telegram/src/main/kotlin/com/github/kotlintelegrambot/network/ApiService.kt#L643)

**Additional context**
Using the same parameters in a direct POST request to the API endpoint (http://api.telegram.org/...), the message gets edited correctly. This validates message ID, chat ID, and file ID.

**Comment by zhtaihao** [comment]
We haven't migrated to version 6.0.4 yet. This bug is observed in 6.0.2.

**Comment by rastiehaiev** [comment]
I face the same problem. When I try to edit message media, Telegram returns 400 with error 
`{"ok":false,"error_code":400,"description":"Bad Request: can't parse input media JSON object"}`. I looked at the code and everything seems to be fine.

**Comment by rastiehaiev** [comment]
I prepared PR with a fix. #215 
@seik Please review, I really need this fix for my bot.

**Comment by zhtaihao** [comment]
@rastiehaiev You may be interested in #214, which also fixes this problem, in a different way than you did. I am currently using that approach and it is working well.

**Comment by rastiehaiev** [comment]
@zhtaihao yeah, I saw it but decided to provide another approach. Approach I used is already currently used in the project, so I did the same (just converted media payload to JSON and store it as a string field). Hope new version will arrive with any of ours fixes so I can use it in my project 🙏.

**Comment by rastiehaiev** [comment]
@zhtaihao my approach doesn't work. Declining PR...

**Comment by zhtaihao** [comment]
This has been fixed in https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/pull/214 @trupizzza 

Closing this issue now.

**Comment by trupizzza** [comment]
> This has been fixed in #214 @trupizzza
> 
> Closing this issue now.

Thanks, I saw this one is merged. May I kindly ask when to wait for new release?

**Comment by zhtaihao** [comment]
Unfortunately I don't work on this project myself. Perhaps @vjgarciag96 can provide insight?

**Comment by vjgarciag96** [comment]
@trupizzza @zhtaihao I just published a new version of the library (v6.0.6) including the fix. It should be available now. Let me know if you are still able to reproduce the issue reported here.

**Comment by trupizzza** [comment]
@vjgarciag96 Thanks for quick fix!
Seems like now (6.0.6) it's not giving the error mentioned before.
Is it ok, that I still cannot add photo to message using this method?

I'm trying to update just-message-with-text-and-inline-keyboard by adding a photo to it, is it possible?
Trying to update text message by editMessageMedia with Photo as InputMedia, I've got following error:

`{"ok":false,"error_code":400,"description":"Bad Request: there is no media in the message to edit"}`

Is there no possibility to update message with no media by adding media via editMessageMedia?
